testsuite/module-publish/src/packages/framework_usecases/sources/resource_groups_example.move

```diff
@@ -0,0 +1,201 @@
+module 0xABCD::resource_groups_example {
+    use std::error;
+    use std::signer;
+    use std::string::{Self, String};
+
+    const EVALUE_TOO_LARGE: u64 = 1;
+
+    #[resource_group(scope = global)]
+    struct ExampleGroup {}
+
+    #[resource_group_member(group = 0xABCD::resource_groups_example::ExampleGroup)]
+    struct ExampleResource0 has key {
+        value: u64,
```

Comment:
Can we add general documentation re: example resource structure / how it's used? e.g. it seems at the moment value isn't used at all?